CallCenterSample/Helpers/TextAnalyticsHelper.cs

```diff
@@ -79,14 +79,28 @@ public static string ApiKeyRegion
             }
         }
 
-        private static void InitializeTextAnalyticsClient()
+        private static string endPoint;
+        public static string EndPoint
         {
-            if (!string.IsNullOrEmpty(ApiKey) && !string.IsNullOrEmpty(ApiKeyRegion))
+            get { return endPoint; }
+            set
             {
-                AnalyticsClient = new TextAnalyticsClient(new ApiKeyServiceClientCredentials())
+                var changed = endPoint != value;
+                endPoint = value;
+                if (changed)
                 {
-                    Endpoint = string.Format("https://{0}.api.cognitive.microsoft.com", ApiKeyRegion)
-                };
+                    InitializeTextAnalyticsClient();
+                }
+            }
+        }
+
+        private static void InitializeTextAnalyticsClient()
+        {
+            if (!string.IsNullOrEmpty(ApiKey) && !string.IsNullOrEmpty(EndPoint) && !EndPoint.Contains("west") &&!EndPoint.Contains("asia"))
```

Comment:
curious why there is exclusion on the regions?

Reply:
I tried with TA services in these regions again, and there were no issues. The exclusion is removed.